src/Tests/Microsoft.Diagnostics.Monitoring.TestCommon/Runners/AppRunner.cs

```diff
@@ -110,10 +110,12 @@ public AppRunner(ITestOutputHelper outputHelper, Assembly testAssembly, int appI
                 AssemblyNameWithoutExtension,
                 tfm);
 
+#pragma warning disable CS0618 // Type or member is obsolete
             _startupHookPath = AssemblyHelper.GetAssemblyArtifactBinPath(
                 testAssembly,
                 "Microsoft.Diagnostics.Monitoring.StartupHook",
                 TargetFrameworkMoniker.Net60);
```

Comment:
I'm assuming we're keeping the startup hook since to ensure we're not depending on apis that don't exist in older frameworks correct?